### PR TITLE
Panelizer update

### PR DESCRIPTION
**Pull request by balsama**
Beta1 of panelizer adds a "save as default" perm which changes the "save as custom" link to just _save_ when you don't have permission to save any other way. So I had to make sure that tests that looked for that link also have the save a default permission.

### CODE REVIEW

File: tests/features/search.feature [hunk 1]
```diff
@@ -15,9 +15,9 @@ Feature: Site search
     And I should not see "Zombie 2"
     And I should see "Zombie 3"
 
-  @javascript
+  @javascript @failing
```
Comment (phenaproxima) [inline tests/features/search.feature]: Should this failing tag be here?

File: tests/features/search.feature [hunk 2]
```diff
@@ -15,9 +15,9 @@ Feature: Site search
...
   Scenario: Indexing and searching for landing pages
-    Given I am logged in as a user with the landing_page_creator,landing_page_reviewer roles
+    Given I am logged in as a user with the landing_page_creator,landing_page_reviewer,layout_manager roles
```
Comment (phenaproxima) [inline tests/features/search.feature]: Why do they get the layout_manager role as well?

File: tests/features/search.feature (hunk 1 above)
Comment (balsama) [reply tests/features/search.feature]: Fixed

File: tests/features/search.feature (hunk 2 above)
Comment (balsama) [reply tests/features/search.feature]: Without it, the save as custom button is just "Save" - and the step definition assumes the link is "Save as custom". Layout manager grants permission to "Save as default" too which changes the custom link back to "Save as custom.

**Review by phenaproxima — APPROVED** [review]

**Pull request merged**